src/app/loans/loans-view/loans-view.component.scss

```
@@ -33,9 +33,51 @@
+// Reason a menu action is blocked, read by screen readers only: the tooltip on
+// the wrapper needs a hover the disabled item never receives.
+.menu-item-reason {
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  margin: -1px;
+  padding: 0;
+  border: 0;
+  overflow: hidden;
+  white-space: nowrap;
+  clip: rect(0 0 0 0);
```

Stylelint (17.14.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)